reference/docs-conceptual/learn/Writing-Portable-Modules.md

```diff
@@ -287,6 +287,48 @@ Example:
 }
 ```
 
+## Dependency on Native Libraries
+
+Modules intended for use across different operating systems or processor architectures may
+depend on a managed library that itself depends on some native libraries.
+
+Prior to PowerShell 7, one would have to have custom code to load the appropriate native
+dll so that the managed library can find it correctly.
+
+With PowerShell 7, native binaries to load are searched in sub-folders within the managed
+library's location following a subset of the [.NET RID Catalog][] notation.
+
+```none
+managed.dll folder
```

Comment:
This chart uses `.dll` extensions for all RID folders, which is wrong. Please use the chart here: https://github.com/PowerShell/PowerShell/pull/11032/files#diff-62e18a07c07b6e0537fd17ccbbd34466R201 instead. The one in the PR description is wrong and was missed in review.

Reply:
Fixed